### DISCUSSION TITLE
Add "Item" as a type for the Global Variable Insights panel

### DISCUSSION THREAD

**Discussion by igorazevedo** [directus/directus#15641] [Feature Requests]
It would be very useful if we could select an item or list of items as a global variable in Insights. Having this feature would allow us to build powerful dashboards/reports.

In the panel configuration:
* Have a new type - Item
* When selected, prompt for the collection to use and if it allows multiple selection

In the panel:
* Have an interface to select existing item or multiple items, similar to what happens when we select items in a relational field

The global variable would store the selected item's primary key or the complete item data (maybe this behavior can be a configuration option too?).

**Comment by azrikahar** [comment]
Coincidentally, this was recently sponsored & implemented in #15452! It is now released in version `9.18.0` 👍 You can find it as `Global Relational Variable` panel, where you will be able to select a single item (similar to M2O interface), or allow multiple item selections (similar to O2M interface), and they will contain the primary key(s).
